views/content/dao-association.sol

```diff
@@ -243,10 +243,9 @@ contract Association is owned, tokenRecipient {
 
         require(quorum >= minimumQuorum); // Check if a minimum quorum has been reached
 
+        p.executed = true;
         if (yea > nay ) {
             // Proposal passed; execute the transaction
-
-            p.executed = true;
```

Comment:
What happens if a proposal is approved but it throws? Should we then change the require into a if clause?

Reply:
If a proposal's transaction throws, should it be allowed to be re-tried until it succeeds? I guess that's partially up to the intent of the DAO; the strictest one would be if it throws, then you need to create a new Proposal with a new transaction (that doesn't throw, presumably) and re-approve that one.

Reply:
The intent was to leave it to be counted up and executed again (if it still has the votes by then). But then I don't think the code was doing this, as if it throws executed won't be set anyway

Reply:
Oh right; the case I was trying to avoid with this change is if the proposal doesn't have enough votes (`yea <= nay`), the `executed` flag doesn't get set (ever), so years down the road, if a proposal didn't have everyone vote (only reached quorum originally), more votes could be tallied (essentially forever; even new members to the DAO could now vote on this old proposal), and the moment it flips to a "Yea" majority, it could be executed.

This change makes it set where once it's voted down, it stays down.